src/python/pants/bin/pants_loader.py

```diff
@@ -46,7 +46,7 @@ def ensure_locale(cls):
 
   @staticmethod
   def determine_entrypoint(env_var, default):
-    return os.environ.get(env_var) or default
+    return os.environ.pop(env_var, default) or default
```

Comment:
Is the `or default` here intended to handle the "empty" entrypoint case? If not, maybe consider removing it and failing fast for an invalid/empty value?

Reply:
a remnant of the prior `get()` default to `None` - removed.